content/api/vision.md

```diff
@@ -5,14 +5,14 @@ title: Vision
 Your webcam is how your robot will see markers.
```

Comment:
Many of the changes in this file are handled in #67, but i'll sort out conflicts there

Reply:
Addressed in https://github.com/sourcebots/docs/pull/67/commits/f969daf472fd3d76a49f083ace10d1353728ff94